cli/pcluster/config/param_types.py

```diff
@@ -0,0 +1,1202 @@
+    are merged together to create CFN parameters.
+    Furthermore, as opposed to SettingsParam, the value can be a comma separated value (e.g. ebs_settings = ebs1,ebs2).
+    """
+
+    def _init_from_file(self, config_parser):
+        """
+        Initialize param value from configuration file.
+
+        :param config_parser: the configparser object from which get the parameter
+        :raise NoSectionError if unable to get the section from config_parser
+        """
+        section_name = _get_file_section_name(self.section_key, self.section_label)
+        try:
+            self.value = config_parser.get(section_name, self.key)
+            if self.value:
+                for section_label in self.value.split(","):
+                    section = self.related_section_type(
+                        self.related_section_map,
+                        self.pcluster_config,
+                        section_label=section_label.strip(),
+                        config_parser=config_parser,
+                        fail_on_absence=True,
+                    )
+                    self.pcluster_config.add_section(section)
+        except NoOptionError:
+            self._init_from_map()
+
+    def _init_from_cfn(self, cfn_params=None, cfn_value=None):
+        """Init ebs section only if there are more than one ebs (the default one)."""
+        labels = []
+        if cfn_params:
+            num_of_ebs = int(get_cfn_param(cfn_params, "NumberOfEBSVol"))
+            if num_of_ebs > 1:
+                for index in range(num_of_ebs):
+                    configured_params = False
+                    # TODO Use the label when will be available
+                    label = "{0}{1}".format(self.related_section_key, str(index + 1))
+                    labels.append(label)
+
+                    # create empty section
+                    related_section_type = self.related_section_map.get("type", Section)
+                    related_section = related_section_type(self.related_section_map, label)
+
+                    for param_key, param_map in self.related_section_map.get("params").items():
+                        cfn_converter = param_map.get("cfn", None)
+                        if cfn_converter:
+
+                            param_type = param_map.get("type", Param)
+                            cfn_value = get_cfn_param(cfn_params, cfn_converter).split(",")[index]
+                            param = param_type(
+                                self.section_key,
+                                self.section_label,
+                                param_key,
+                                param_map,
+                                self.pcluster_config,
+                                cfn_value=cfn_value,
+                            )
+                            related_section.add_param(param)
+
+                            if param.value != param_map.get("default", None):
```

Comment:
what if customer configures multiple ebs volumes all with default paramenters (if possible)?

Reply:
It is not possible, the customer must at least specify the `shared_dir`.